README.md

```diff
@@ -15,18 +15,18 @@ operator-sdk run --local
 ## Using an init container to load graph data
 
 The UpdateService graph data is loaded from an init container. Before deploying 
-the updateservice-operator, you will need to [build and push an init container containing the graph data](docs/graph-data-init-container.md).
+the update-service-operator, you will need to [build and push an init container containing the graph data](docs/graph-data-init-container.md).
```

Comment:
can we just say "the operator" here?

Reply:
Yeah, agree

Reply:
fixed in https://github.com/openshift/cincinnati-operator/commit/f99ff7df60ccd1ffae42fcb1a4cd8b6f3d78027f